submitted_models/ctu_cras_norlab_lily_sensor_config_1/urdf/sensors/cameras.xacro

```diff
@@ -65,6 +65,15 @@
                         <mean>0.0</mean>
                         <stddev>0.007</stddev>
                     </noise>
+                    <lens>
+                        <intrinsics>
+                            <fx>1020.51</fx>
```

Comment:
What formula did you use to calculate 1020.51?

I followed the standard `fx = fy = width / ( 2 * tan (hfov / 2 ) )` to reach 1029.47

Reply:
I apparently used FOV 86.5°. The measured FOV is 86.8° (see MARV robot PR).

Reply:
Good point.  Can you propagate the documentation for the sensor FoV over to Lily's `specifications.md` and adjust the sensor FoV on Lily to be consistent with the camera intrinsics?

Reply:
Ok, FOV is fixed.